### ISSUE TITLE
[FEATURE REQUEST] Change models directory

### ISSUE THREAD

**Issue by bobcate** [Loewen-Hob/rembg-comfyui-node#5]
Requesting the ability to change the directory where the models are saved.

**Comment by Loewen-Hob** [comment]
All models are downloaded and saved in the user home folder in the **.u2net** directory.

**Comment by bobcate** [comment]
Yeah, that's what it says in the readme.
What I'm suggesting is being able to specify where they are stored.
Like, add a `modelsPath = /path/to/models/` config.

**Comment by 3blackbar** [comment]
I agree, would be nice to point to a folder with models collection, most of us have multiple guis installed, we dont want multiple clones of the models on the same disk

**Comment by Loewen-Hob** [comment]
Everyone is right! this is a point I hadn't considered before, I'll update this feature quickly when I have time, thanks for the suggestions!

**Comment by dikoweii** [comment]
请问主目录是那个位置
